### ISSUE TITLE
Neural Network Optimization

### ISSUE THREAD

**Issue by chfor183** [chfor183/data_science_articles#15]
## TL;DR

Yes !

## Key Takeaways
- 1
- 2

## Useful Code Snippets
```
function test() {
  console.log("notice the blank line before this function?");
}
```

## Articles/Ressources
https://towardsdatascience.com/neural-networks-parameters-hyperparameters-and-optimization-strategies-3f0842fac0a5
https://towardsdatascience.com/hyperparameters-optimization-526348bb8e2d
https://towardsdatascience.com/how-to-increase-the-accuracy-of-a-neural-network-9f5d1c6f407d
https://towardsdatascience.com/hyper-parameter-tuning-and-model-selection-like-a-movie-star-a884b8ee8d68
